app/templates/me/index.hbs

```diff
@@ -33,8 +33,8 @@
         If you want to use package commands from the command line, you will need to
         login with <code>cargo login (token)</code> using one of the tokens listed below. <br/>
```

Comment:
Could you make this 2 separate paragraphs instead? Would make it easier to read :)